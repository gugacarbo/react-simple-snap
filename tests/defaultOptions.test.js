const defaultOptions = require("../src/defaultOptions");

test("defaultOptions", () => {
  expect(defaultOptions).toMatchSnapshot();
});
