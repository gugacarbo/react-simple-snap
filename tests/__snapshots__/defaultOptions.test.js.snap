// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`defaultOptions 1`] = `
{
  "asyncScriptTags": false,
  "cacheAjaxRequests": false,
  "concurrency": 4,
  "crawl": true,
  "destination": null,
  "exclude": [],
  "externalServer": false,
  "fixInsertRule": true,
  "fixWebpackChunksIssue": "CRA1",
  "http2PushManifest": false,
  "ignoreForPreload": [
    "service-worker.js",
  ],
  "include": [
    "/",
  ],
  "inlineCss": false,
  "minifyCss": {},
  "minifyHtml": {
    "collapseBooleanAttributes": true,
    "collapseWhitespace": true,
    "decodeEntities": true,
    "keepClosingSlash": true,
    "sortAttributes": true,
    "sortClassName": false,
  },
  "optionsFile": ".snap.json",
  "port": 45678,
  "preconnectThirdParty": true,
  "preloadImages": false,
  "publicPath": "/",
  "puppeteer": {
    "cache": true,
    "headless": true,
  },
  "puppeteerArgs": [],
  "puppeteerExecutablePath": undefined,
  "puppeteerIgnoreHTTPSErrors": false,
  "removeBlobs": true,
  "removeScriptTags": false,
  "removeStyleTags": false,
  "saveAs": "html",
  "screenshot": false,
  "sitemap": false,
  "skipThirdPartyRequests": false,
  "source": "dist",
  "sourceMaps": true,
  "userAgent": "react-simple-snap",
  "viewport": {
    "height": 850,
    "width": 480,
  },
  "waitFor": false,
}
`;
